src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -11,17 +11,18 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "goto_instrument_parse_options.h"
 
-#include <fstream>
-#include <iostream>
-#include <memory>
-
 #include <util/exception_utils.h>
 #include <util/exit_codes.h>
 #include <util/json.h>
+#include <util/options.h>
```

Comment:
Do we need this to include `set` and `string`?

Reply:
this is needed for all optionst declarations contained in the file

```
../src/goto-instrument/goto_instrument_parse_options.cpp:1001:12: error: variable has incomplete type 'optionst'
  optionst options;
           ^
../src/goto-programs/restrict_function_pointers.h:34:7: note: forward declaration of 'optionst'
class optionst;
      ^
../src/goto-instrument/goto_instrument_parse_options.cpp:1272:14: error: variable has incomplete type 'optionst'
    optionst c_object_factory_options;
             ^
../src/goto-programs/restrict_function_pointers.h:34:7: note: forward declaration of 'optionst'
class optionst;
      ^
../src/goto-instrument/goto_instrument_parse_options.cpp:1291:14: error: variable has incomplete type 'optionst'
    optionst c_object_factory_options;
             ^
../src/goto-programs/restrict_function_pointers.h:34:7: note: forward declaration of 'optionst'
class optionst;
```